### PR TITLE
feat(pie-monorepo): DSW-2056 pie webc integration next13

### PR DESCRIPTION
**Pull request by kevinrodrigues**
Please click the `Preview` tab and select a different PR template if your change isn't for an                                            existing web component.

[PIE Docs Template](?expand=1&template=docs_template.md)
[New Web Component Template](?expand=1&template=new_component_template.md)

═══════════════════════════════════════════════════════════

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code

### CODE REVIEW

File: apps/examples/wc-next13/.gitignore
Comment (xander-marjoram) [inline apps/examples/wc-next13/.gitignore:1]: Was this an automated update?

File: apps/examples/wc-next13/package.json [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
     "@types/react-dom": "18.3.0",
-    "eslint": "8.37.0",
-    "eslint-config-next": "13.5.5",
-    "typescript": "5.4.5"
-  },
-  "volta": {
-    "extends": "../../../package.json"
-  },
-  "installConfig": {
-    "hoistingLimits": "workspaces"
```
Comment (xander-marjoram) [inline apps/examples/wc-next13/package.json:32]: I think it's probably important to keep these sections

**Review by sebastienrocks — DISMISSED** [review]

File: apps/examples/wc-next13/.gitignore
Comment (kevinrodrigues) [reply apps/examples/wc-next13/.gitignore:1]: From a new vite/next install (deleted the old folder and started up a new project). There were errors in the old one similar to the react18 & 17 folders.

File: apps/examples/wc-next13/package.json [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
+  "type": "module",
   "scripts": {
-    "dev": "NODE_OPTIONS=--conditions=development next dev",
-    "build:examples": "yarn build-static",
-    "build-static": "next build && next export -o build",
-    "start": "next start",
-    "lint:examples": "next lint"
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
```
Comment (xander-marjoram) [inline apps/examples/wc-next13/package.json:9]: `--report-unused-disable-directives` is very cool! Is it possible to add that to the config file?

**Review by xander-marjoram — APPROVED** [review]

File: apps/examples/wc-next13/package.json (hunk 2 above)
Comment (kevinrodrigues) [reply apps/examples/wc-next13/package.json:9]: Sorry missed this comment. Do you mean in the lint config file?

Comment (xander-marjoram) [reply apps/examples/wc-next13/package.json:9]: Sorry, yes, the eslintrc

Comment (kevinrodrigues) [reply apps/examples/wc-next13/package.json:9]: Seems to be part of eslint & CLI so i dont think this can be moved unfortunately.

**Review by sebastienrocks — APPROVED** [review]

**Pull request merged**